lib/gringotts/money.ex

```diff
@@ -8,38 +8,112 @@ defprotocol Gringotts.Money do
   If your application is already using a supported Money library, just pass in
   the Money struct and things will work out of the box.
 
-  Otherwise, just wrap your `amount` with the `currency` together in a `Map` like so,
-      money = %{amount: Decimal.new(2017.18), currency: "USD"}
+  Otherwise, just wrap your `amount` with the `currency` together in a `Map`
+  like so,
 
-  and the API will accept it (as long as the currency is valid [ISO 4217 currency
-  code](https://www.iso.org/iso-4217-currency-codes.html)).
+      price = %{amount: Decimal.new(20.18), currency: "USD"}
```

Comment:
What happened to `value` suggestion for representing amount map. I remember we had a discussion about it internally.

eg:
`%{value: Decimal.new(20.18), currency: "USD"}`

cc @aviabird/all